mozilla/fx-private-relay
# Add the Credential Manager integration announcement

# New feature description

This adds a news announcement about the integration with Credential Manager.

# Screenshot (if applicable)

See preview site.

# How to test

Set your computer clock to December 14th or later, then open the dashboard in Firefox. You should see the announcement.

# Checklist

- [x] All acceptance criteria are met. - Sort of, there's no mention of the icon being turned off, but we won't do it and it wasn't in the designs either.
- [ ] User Story has been updated (if needed) to match changes made during the development process. - I added one case: hiding the announcement if not using Firefox. Still need to get approval about that.
- [x] User Story has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
- [x] l10n changes have been submitted to the l10n repository, if any. - https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/115
